.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: lf-sample-repository-api-dotnet-srv-CI
+
+on:
+  push:
+    branches: [ '\d+.x' ]
+  pull_request:
+    branches: [ '\d+.x' ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build-n-run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: |
+          3.1.x
+          6.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build --no-restore
+
+    - name: Run sample
+      env:
+        ACCESS_KEY: ${{ secrets.DEV_CA_PUBLIC_USE_INTEGRATION_TEST_ACCESS_KEY }}
+        SERVICE_PRINCIPAL_KEY:  ${{ secrets.DEV_CA_PUBLIC_USE_TESTOAUTHSERVICEPRINCIPAL_SERVICE_PRINCIPAL_KEY }}

```

Comment:
Is this name correct? doesn't need _ characters to make it more readable?

Reply:
if you are referring the variables under env, i believe the variables underwent a name change (changed it to a more generic name) a couple days ago for all the C# and JS/TS projects

Reply:
I mean DEV_CA_PUBLIC_USE_TESTOAUTHSERVICEPRINCIPAL_SERVICE_PRINCIPAL_KEY 
it's too long, and seems a few underscores are omitted. Regardless of length, this would be better:
DEV_CA_PUBLIC_USE_TEST_OAUTH_SERVICE_PRINCIPAL_SERVICE_PRINCIPAL_KEY 

By the way, SERVICE_PRINCIPAL is used twice.

Reply:
yea so those secret.variables come from this section in the settings tab below, unfortunately i do not have the permissions to actually edit/change the variables names or values, I think @jason-chan-lf has the permission to do that  

![image](https://user-images.githubusercontent.com/103658932/173084729-19293e50-9014-4bfd-a115-8f0316325aec.png)

Reply:
Oh, I got it now. The part in the circle is ok. So, I will approve the pr. I think the secret names is beyond this pr's scope.

Reply:
alright thanks!